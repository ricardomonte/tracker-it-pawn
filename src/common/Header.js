import React from 'react';
import TrackerStyle from '../styles/Header.module.css';

const Header = () => (
  <div className={TrackerStyle.title}>
    <h1 className={TrackerStyle.text}>Track-It-Pawn</h1>
  </div>
);

export default Header;
