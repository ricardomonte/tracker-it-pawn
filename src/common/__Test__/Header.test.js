import React from 'react';
import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import Header from '../Header';

test('Content from About matches snapshot', () => {
  const content = render(<Header />);
  expect(content.container).toMatchSnapshot();
  expect(screen.getByText('Track-It-Pawn')).toBeInTheDocument();
});
