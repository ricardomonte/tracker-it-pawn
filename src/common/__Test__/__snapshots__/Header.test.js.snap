// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content from Header matches snapshot 1`] = `
<div>
  <div
    class="title"
  >
    <h1
      class="text"
    >
      Track-It-Pawn
    </h1>
  </div>
</div>
`;
